tests/testthat/test-understat.R

```diff
@@ -47,3 +48,6 @@ test_that("understat_team_stats_breakdown() works", {
   expect_gt(nrow(team_stats), 0)
 })
 
+test_that("understat_league_season_shots() works", {
```

Comment:
you can just delete this. `check_league_name` is already directly tested and indirectly tested

Reply:
Done :+1: .